src/ontodia/workspace/workspace.ts

```diff
@@ -26,6 +25,8 @@ export interface Props {
     isDiagramSaved?: boolean;
     hideTutorial?: boolean;
     viewOptions?: DiagramViewOptions;
+    leftPanelInitiallyOpen?: boolean;
```

Comment:
Set default value `true` for `leftPanelInitiallyOpen` (in `Workspace.defaultProps`).